e2e-tests/cypress/fixtures/customKibanaConfig.yml

```
@@ -0,0 +1,65 @@
+        document.body.appendChild(div);
+      };
+      if (window.ROR_METADATA.enrichedData) {
+        const div = document.createElement('div');
+        div.setAttribute('data-testid', 'metadata-enriched-data');
+        div.textContent = window.ROR_METADATA.enrichedData;
+        document.body.appendChild(div);
+      };"
+  custom_middleware_inject: "async function customMiddleware(req, res, next) {
+      const metadata =
+        req.rorRequest && req.rorRequest.getIdentitySession() && req.rorRequest.getIdentitySession().metadata;
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)